app/models/player.rb

```diff
@@ -11,10 +11,16 @@ def buy(card_name)
   end
 
   def deck
-    cards.where(discarded: false)
+    cards.where(discarded: false).order(:deck_index)
   end
 
   def discard
```

Comment:
These are logical methods, l I'll likely add an order by index to the discards for oyr end turn endpoint